# 下载文献


在制作 Slides 的过程中，我需要搜索很多文献。由于时间和能力的限制，我无法对每篇文献进行深入的解读和总结。为此，我使用如下流程加速我对这些文献的理解：

1. 使用我编写的 Stata 命令 [getiref](https://www.lianxh.cn/details/1382.html) 获得引文信息，并下载 PDF 文档到指定路径。
2. 将这些 PDF 文档 (一次一篇或同主题的多篇) 发送给 AI 助手 (我主要使用 ChatGPT 和豆包)，让它们帮我解读论文的要点。我会酌情追问，并阅读原文中的相关内容。如有必要，我会把这些内容发送给 AI 助手，让它解读，或提供更容易理解的公式、实例、Stata/Python 代码等。

本文档主要用于下载文献到 **./refs** 目录下。

## 用 getiref 下载引文

你可以直接在 Stata 命令窗口中执行这些命令，也可以在 dofile 中执行。此处，我已经配置好了 Python + Jupyter Notebook 环境 ([配置方法](https://book.lianxh.cn/ds/body/01_1_install-Python-Anocanda.html))，因此，我直接在 Jupyter Notebook 中执行这些命令。


In [5]:

* 安装 getiref
//ssc install getiref, replace

* 设置下载路径
global refpath "D:\github_lianxh\research_with_AI\refs"
cd "$refpath"

D:\github_lianxh\research_with_AI\refs


In [2]:
getiref 2507.10933 , m arxiv pdf

  
Erdem, O., & Ashok, R. P. (2025). Artificial Finance: How AI Thinks About Money
>  (Version 1). arXiv.
    Link (rep)    PDF    Google    

Erdem, O., & Ashok, R. P. (2025). Artificial Finance: How AI Thinks About Money
>  (Version 1). arXiv. [Link](https://doi.org/10.48550/arXiv.2507.10933) (rep),
>  [PDF](https://arxiv.org/pdf/2507.10933.pdf), [Google](<https://scholar.googl
> e.com/scholar?q=Artificial Finance: How AI Thinks About Money (Version 1)>).
        PDF:   dir    view_online     Open
 
Tips: Text is on clipboard. Press 'Ctrl+V' to paste, ^-^


执行完上述命令后，引文信息已经被自动粘贴在剪切板上，直接按快捷键 `Ctrl + V` 即可粘贴到 Jupyter Notebook 中。同时，PDF 文档已经下载到指定路径。

### 下载好引文信息

- Erdem, O., & Ashok, R. P. (2025). Artificial Finance: How AI Thinks About Money (Version 1). arXiv. [Link](https://doi.org/10.48550/arXiv.2507.10933) (rep), [PDF](https://arxiv.org/pdf/2507.10933.pdf), [Google](<https://scholar.google.com/scholar?q=Artificial Finance: How AI Thinks About Money (Version 1)>).

## 其他文献

In [6]:
getiref 10.1016/j.accinf.2024.100715, m 

  
Dong, M. M., Stratopoulos, T. C., & Wang, V. X. (2024). A scoping review of Cha
> tGPT research in accounting and finance. International Journal of Accounting 
> Information Systems, 55, 100715.
    Link    PDF    Google    

Dong, M. M., Stratopoulos, T. C., & Wang, V. X. (2024). A scoping review of Cha
> tGPT research in accounting and finance. International Journal of Accounting 
> Information Systems, 55, 100715. [Link](https://doi.org/10.1016/j.accinf.2024
> .100715), [PDF](http://sci-hub.ren/10.1016/j.accinf.2024.100715), [Google](<h
> ttps://scholar.google.com/scholar?q=A scoping review of ChatGPT research in a
> ccounting and finance>).
 
Tips: Text is on clipboard. Press 'Ctrl+V' to paste, ^-^


- Dong, M. M., Stratopoulos, T. C., & Wang, V. X. (2024). A scoping review of ChatGPT research in accounting and finance. International Journal of Accounting Information Systems, 55, 100715. [Link](https://doi.org/10.1016/j.accinf.2024.100715), [PDF](http://sci-hub.ren/10.1016/j.accinf.2024.100715), [Google](<https://scholar.google.com/scholar?q=A scoping review of ChatGPT research in accounting and finance>).

In [7]:
getiref      10.1016/j.futures.2024.103453   , m

  
Renkema, M., & Tursunbayeva, A. (2024). The future of work of academics in the 
> age of Artificial Intelligence: State-of-the-art and a research roadmap. Futu
> res, 163, 103453.
    Link    PDF    Google    

Renkema, M., & Tursunbayeva, A. (2024). The future of work of academics in the 
> age of Artificial Intelligence: State-of-the-art and a research roadmap. Futu
> res, 163, 103453. [Link](https://doi.org/10.1016/j.futures.2024.103453), [PDF
> ](http://sci-hub.ren/10.1016/j.futures.2024.103453), [Google](<https://schola
> r.google.com/scholar?q=The future of work of academics in the age of Artifici
> al Intelligence: State-of-the-art and a research roadmap>).
 
Tips: Text is on clipboard. Press 'Ctrl+V' to paste, ^-^


- Renkema, M., & Tursunbayeva, A. (2024). The future of work of academics in the age of Artificial Intelligence: State-of-the-art and a research roadmap. Futures, 163, 103453. [Link](https://doi.org/10.1016/j.futures.2024.103453), [PDF](https://ris.utwente.nl/ws/portalfiles/portal/489029873/1-s2.0-S0016328724001368-main.pdf), [Google](<https://scholar.google.com/scholar?q=The future of work of academics in the age of Artificial Intelligence: State-of-the-art and a research roadmap>).

In [9]:
getiref       10.2139/ssrn.4736003  , m

  
Giesecke, O. (2024). AI at the Frontier of Economic Research. SSRN Electronic J
> ournal.
    Link        Google    

Giesecke, O. (2024). AI at the Frontier of Economic Research. SSRN Electronic J
> ournal. [Link](https://doi.org/10.2139/ssrn.4736003), [Google](<https://schol
> ar.google.com/scholar?q=AI at the Frontier of Economic Research>).
 
Tips: Text is on clipboard. Press 'Ctrl+V' to paste, ^-^


- Giesecke, O. (2024). AI at the Frontier of Economic Research. SSRN Electronic Journal. [Link](https://doi.org/10.2139/ssrn.4736003), [-PDF-](https://papers.ssrn.com/sol3/Delivery.cfm/SSRN_ID4736003_code2051012.pdf?abstractid=4736003&mirid=1&type=2), [Google](<https://scholar.google.com/scholar?q=AI at the Frontier of Economic Research>).
